README.md

```diff
@@ -19,3 +19,146 @@ files. CMake developers document their functions by immediately proceeding the
 function with a comment block written in reStructuredText (reST). CMakeDoc will
 extract the comment blocks verbatim and create `*.rst` files which can be
 included in a normal Sphinx documentation set-up.
+
+## Installing
+
+#### With PIP / PyPI
+Run:
+```console
+foo@bar:~$ sudo pip3 install cmakedoc
+```
+
+#### Manually
+Run the following commands one at a time:
+
+```console
+foo@bar:~$ git clone github.com/CMakePP/CMakeDoc.git
+foo@bar:~$ cd CMakeDoc/
+foo@bar:~/CMakeDoc$ sudo pip3 install . #If pip installed
+foo@bar:~/CMakeDoc$ sudo python3 setup.py install #If pip not installed
+```
+
+## Usage
+For each CMake function or variable that you would like to document, prepend it with a block doc-comment.
+A block doc-comment begins with `#[[[` and ends with `#]]`.
+
+Then run `cmakedoc` on your CMake files, outputting to a directory of your choosing. The help text is printed below for reference:
+```
+Usage: cmakedoc [-h] [-o OUTPUT] [-r] file [file ...]
+
+positional arguments:
+  file                  CMake file to generate documentation for. If
+                        directory, will generate documentation for all *.cmake
+                        files (case-insensitive)
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -o OUTPUT, --output OUTPUT
+                        Directory to output generated RST to. If not specified
+                        will print to standard output. Output files will have
+                        the original filename with the cmake extension
+                        replaced by .rst
+  -r, --recursive       If specified, will generate documentation for all
+                        subdirectories of specified directory recursively
+```
+
+### Example
+example.cmake:
```

Comment:
The example needs more prose. For example, explain that `example.cmake` is the file being documented and that it already includes the special comments.

Reply:
Added some explanation